native/dlasq2.go

```diff
@@ -0,0 +1,326 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/lapack"
+)
+
+// Dlasq2 computes all the eigenvalues of the symmetric positive
```

Comment:
"Dlasq2 computes all the eigenvalues of the symmetric positive definite tridiagonal matrix associated with the qd array Z to high relative accuracy~~ are computed to high relative accuracy~~, in the absence of denormalization, underflow and overflow."

The original FORTRAN comment is not English.